docs/libvirt-host-setup.md

```diff
@@ -1,33 +1,85 @@
+    - [Password less access](#password-less-access)
+    - [Verification](#verification)
+  - [RHCOS and RHEL 8.X Images for OpenShift](#rhcos-and-rhel-8x-images-for-openshift)
+    - [Download the RHEL Qcow2 image](#download-the-rhel-qcow2-image)
+    - [Customize the RHEL Qcow2 image](#customize-the-rhel-qcow2-image)
+    - [Download the RHCOS Qcow2 image](#download-the-rhcos-qcow2-image)
 
+<!-- /TOC -->
+## Libvirt Configuration
 
-## Install and Configure Libvirt
+The following steps are required to configure Libvirt to work with the automation code
+### Install and Configure Libvirt
 
 Please follow the steps given at [Install and Enable section](https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#install-and-enable-libvirt)
 
-## Enable IP Forwarding
+### Enable IP Forwarding
 
 Please follow the steps given at [IP forwarding section](https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#enable-ip-forwarding)
 
-## Accept TCP connections
+### Accept TCP connections
 
 Please follow the steps given at [Accept TCP connections section](https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#for-permanently-running-libvirt-daemon)
 
-## Allow Firewall for libvirt connections
+### Allow Firewall for libvirt connections
 
 Please follow the steps given at [Firewalld section](https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#firewalld)
 
-## Password less access
+### Password less access
 
 Follow below steps to add the public key of the Terraform client machine user to the authorized list for password-less access.
 1. Copy ~/.ssh/id_rsa.pub contents from Terraform client machine.
 2. Append the public key as copied above to ~/.ssh/authorized_keys on the KVM host.
 
-## Verificcation
+### Verification
 
 You can verify TCP connections to the kvm host using an example `virsh` command given below.
 ```
 virsh --connect qemu+tcp://<host_name_or_ip>/system --readonly
 ```
+
+
+## RHCOS and RHEL 8.X Images for OpenShift
+
+You'll need to have the RedHat CoreOS (RHCOS) and RHEL 8.2 (or later) image on the libvirt host.
+RHEL 8.x image is used by bastion node, and RHCOS image is used for boostrap, master and worker nodes.
+### Download the RHEL Qcow2 image
+
+- Login to the RedHat portal and click on the [Downloads](https://access.redhat.com/downloads) tab.
+- Click on the product [Red Hat Enterprise Linux 8](https://access.redhat.com/downloads/content/479/) from the list.
+- In the "Product Variant" drop-down select "Red Hat Enterprise Linux for Power, little endian".
+- In the "Version" drop-down select the version of RHEL you want to download. For example, will use the latest as 8.2.
+- In the "Product Software" tab, click on "Download Now" button adjacent to "Red Hat Enterprise Linux 8.2 Update KVM Guest Image".
+- Save the qcow2 image on your machine, place it in the Libvirt images directory.
```

Comment:
No need of placing the file on libvirt host or images directory. This step was given for customizing the qcow2 file in later step. This can be placed anywhere on the TF client machine or hosted on a HTTP server.